src/pkg/selectors/onedrive.go

```diff
@@ -376,20 +378,27 @@ func (c oneDriveCategory) isLeaf() bool {
 // Example:
 // [tenantID, service, userPN, category, folder, fileID]
 // => {odFolder: folder, odFileID: fileID}
-func (c oneDriveCategory) pathValues(repo path.Path, ent details.DetailsEntry) map[categorizer][]string {
+func (c oneDriveCategory) pathValues(
+	repo path.Path,
+	ent details.DetailsEntry,
+) (map[categorizer][]string, error) {
+	if ent.OneDrive == nil {
+		return nil, clues.New("no OneDrive ItemInfo in details")
+	}
+
 	// Ignore `drives/<driveID>/root:` for folder comparison
 	rFld := path.Builder{}.Append(repo.Folders()...).PopFront().PopFront().PopFront().String()
 
 	result := map[categorizer][]string{
 		OneDriveFolder: {rFld},
-		OneDriveItem:   {repo.Item(), ent.ShortRef},
+		OneDriveItem:   {ent.OneDrive.ItemName, ent.ShortRef},
```

Comment:
`ItemName` [looks like](https://github.com/alcionai/corso/commit/c4e5915983e2098fd4de01131a400bacf222d585#diff-cd4ad81a441fe5b1a43835fb95436f2a797ad246312beb00704340a57d810c65R126) it's been populated from the very beginning so if it's empty it's probably an error somewhere rather than a legacy feature